backend/src/main/java/com/festago/infrastructure/RandomVerificationCodeProvider.java

```diff
@@ -0,0 +1,25 @@
+package com.festago.infrastructure;
+
+import static java.util.stream.Collectors.collectingAndThen;
+import static java.util.stream.Collectors.joining;
+
+import com.festago.domain.VerificationCode;
+import com.festago.domain.VerificationCodeProvider;
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.stream.IntStream;
+import org.springframework.stereotype.Component;
+
+@Component
+public class RandomVerificationCodeProvider implements VerificationCodeProvider {
+
+    private static final Random random = ThreadLocalRandom.current();
+    private static final int CODE_LENGTH = 6;
+
+    @Override
+    public VerificationCode provide() {
+        return IntStream.range(0, CODE_LENGTH)
+            .mapToObj(i -> String.valueOf(random.nextInt(10)))
+            .collect(collectingAndThen(joining(), VerificationCode::new));
+    }
```

Comment:
ThreadLocalRandom 클래스를 사용하는 목적은 스레드 별로 랜덤한 값을 생성하기 위함인데, static 영역으로 ThreadLocalRandom 변수를 선언하는 것은 일반 Random 클래스를 사용하는 것과 다를 것이 없어보이네요!

따라서 다음과 같이 수정해야 할 듯 싶습니다.

```suggestion
    private static final int CODE_LENGTH = 6;

    @Override
    public VerificationCode provide() {
        Random random = ThreadLocalRandom.current();
        return IntStream.range(0, CODE_LENGTH)
            .mapToObj(i -> String.valueOf(random.nextInt(10)))
            .collect(collectingAndThen(joining(), VerificationCode::new));
    }
```